api/indexer.oas2.json

```diff
@@ -2003,6 +2101,13 @@
       "in": "path",
       "required": true
     },
+    "sender-address": {
```

Comment:
we have `address-role=sender` for this.

Reply:
this is one for logs only that are in app call txns with Sender only